app/models/bulkrax/oai_dc_entry.rb

```diff
@@ -6,8 +6,8 @@ class OaiDcEntry < OaiEntry
     matcher 'creator', split: true
     matcher 'date', from: ['date'], split: true
     matcher 'description'
-    matcher 'format_digital', from: ['format_digital', 'format'], parsed: true
-    matcher 'identifier', from: ['identifier'], if: ->(parser, content) { content.match(/http(s{0,1}):\/\//) }
+    matcher 'format_digital', from: %w[format_digital format], parsed: true
```

Comment:
=-p 
%w is so trendy 
/end curmudgeon mode

Reply:
Sorry! (rubocop again, I can never remember the trendy syntax)